### PR TITLE
safe variance

### PR DESCRIPTION
**Pull request by vitskvara**
Maybe we could solve #3  simply like this? I have ran into this issue with conv nets as well.

### CODE REVIEW

**Comment by nmheim** [comment]
Fine with me. The other mean functions would just need to be changed as well

**Comment by vitskvara** [comment]
I added the constant also to `cmean_gaussian`, is there any other place it needs to go to?

File: src/cmeanvar_gaussian.jl [hunk 1]
```diff
@@ -54,9 +54,9 @@ function mean_var(p::CMeanVarGaussian{T,ScalarVar}, z::AbstractArray) where T
     end
 
     μ = ex[1:end-1,:]
-    σ = ex[end:end,:] .* fill!(similar(μ, 1, size(μ,2)), 1)
+    σ = ex[end:end,:]
```
Comment (nmheim) [inline src/cmeanvar_gaussian.jl:57]: nice! :)

**Comment by nmheim** [comment]
It should also go in the normal `Gaussian` I think :)

**Comment by nmheim** [comment]
thank you! :)

**Pull request merged**